package/MDAnalysis/transformations/translate.py

```diff
@@ -138,4 +140,170 @@ def wrapped(ts):
         return ts
     
     return wrapped
-    
\ No newline at end of file
+
+    
+def center_in_plane(ag, plane, coordinate=0, origin=None, center='geometry', wrap=False):
+    """
+    Translates the coordinates of a given :class:`~MDAnalysis.coordinates.base.Timestep`
+    instance so that the center of geometry/mass of the given :class:`~MDAnalysis.core.groups.AtomGroup`
+    is centered on a plane. This plane can only be parallel to the x, y or z planes. 
+    
+    Example
+    -------
+    
+    .. code-block:: python
+    
+        ag = u.residues[1].atoms
+        ts = MDAnalysis.transformations.center_in_plane(ag,'x', 1, center='mass')(ts)
+    
+    Parameters
+    ----------
+    ag: AtomGroup
+        atom group to be centered on the unit cell.
+    plane: str
```

Comment:
This is still an issue.

Reply:
Centering on xz,xy and yz planes is not trivial. Since these planes can more than 1 axis, the center of these planes is the center of the cross-section area formed by these planes in the unit cell.
Currently, the code only works with planes that are parallel to x=0, y=0 or z=0.

Reply:
The way I see it, assuming a orthogonal box, centering on XY means you do not change the Z coordinate. It is always tricky to have a mental picture of an arbitrary triclinic box, but assuming the lower triangle of the box matrix is 0, then it should work the same. If the triangle is not 0, then it is trickier as you do have to change Z; but we can make sure it is not the case and fail if it is.

Reply:
okay I see I'm confuse. XY is the same as z=0, XZ is y=0 and YZ is x=0. Ok. I was talking about non-perpendicular planes, sorry. Okay, I simply change the strings to match.